### DISCUSSION TITLE
CSV import support

### DISCUSSION THREAD

**Discussion by vitalics** [oven-sh/bun#5347] [Q&A]
Bun already have `.json`, `.toml`, `.yaml` file extensions. My project uses `.csv` file importing. And I wondering - is it possible to import without reading and use libraries to parse convert into JS objects?

**Comment by bitdom8** [comment]
is there a way? Thanks

**Comment by bitdom8** [comment]
this is the way

```
import { file } from "bun";

let bstocks2here = await Bun.file(bstocks2).text()
const watcher = watch(import.meta.dir);
for await (const event of watcher) {
  console.log(`Detected ${event.eventType} in ${event.filename}`);
  // server.stop()
  // server
  bstocks2here = await Bun.file(bstocks2).text()
}
```

**Comment by truongan07** [comment]
> this is the way
> 
> ```
> import { file } from "bun";
> 
> let bstocks2here = await Bun.file(bstocks2).text()
> const watcher = watch(import.meta.dir);
> for await (const event of watcher) {
>   console.log(`Detected ${event.eventType} in ${event.filename}`);
>   // server.stop()
>   // server
>   bstocks2here = await Bun.file(bstocks2).text()
> }
> ```

That way will not use --hot flag or --watch flag if content of bstocks2 was changed. You can do this : 

```
import csv from './data.csv' with { type: 'text' };
```

**Reply by bitdom8** [reply]
Thank you sir, ahh!

**Comment by lodembeep** [comment]
Can someone make a package out of this ? and expose a few APIs like using CSV headers for object property names ?
